app/main/views/marketplace.py

```diff
@@ -68,7 +68,7 @@ def terms_and_conditions():
 def get_brief_by_id(framework_slug, brief_id):
```

Comment:
We should change `framework_slug` to `framework_framework` or similar, just so we don't get confused between a individual framework and a framework framework. Unfortunately this is all down to poor naming of framework_framework but that's a bigger problem...